### PR TITLE
Update the changelog for v5.0-beta.2

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

**Review by aljazerzen — APPROVED** [review]

File: docs/changelog/5_x.rst [hunk 1]
```diff
@@ -4,13 +4,12 @@ v5.0
 
 :edb-alt-title: EdgeDB v5
 
-To play with the new features, make sure to specify version 5.0 when 
-initializing the project as pre-release versions are not considered stable 
+To play with the new features, make sure to specify version 5.0 when
+initializing the project as pre-release versions are not considered stable
 and will not be automatically suggested:
```
Comment (raddevon) [inline docs/changelog/5_x.rst:9]:
```suggestion
and will not be automatically suggested:

```
Build fails without a line before the `code-block`.

File: docs/changelog/5_x.rst [hunk 2]
```diff
@@ -173,6 +172,9 @@ manage "feature" branches:
    then merge the EdgeDB branch as well (or rename the "feature" EdgeDB branch
    to "main", if the old branch is no longer needed).
 
+We've added ``edgedb branch`` commands to our CLI as well that create, copy,
+rename, drop, and rebase EdgeDB branches.
```
Comment (raddevon) [inline docs/changelog/5_x.rst:176]:
```suggestion
We've added :ref:`edgedb branch commands <ref_cli_edgedb_branch>` to our CLI as
well that create, copy, rename, drop, and rebase EdgeDB branches.
```
In making this suggestion, I've noticed the visual links on the target page are wrong, so I'll fix that in another PR. (I had updated the hidden TOC before merging but not the visible links. 😞)

**Review by raddevon — APPROVED** [review]
One critical change and one suggestion, and we should be good!

File: docs/changelog/5_x.rst (hunk 2 above)
Comment (raddevon) [reply docs/changelog/5_x.rst:176]: Here's that PR to fix the branches command links: https://github.com/edgedb/edgedb/pull/7092

**Pull request merged**